### PR TITLE
Advanced search options

### LINKED ISSUE #2: Advanced search option to UI
Currently the UI is using the simple search notation of `chr : pos ref > alt|vt`. Add an optional advanced search panel that can be used to specify all possible request parameters: https://github.com/CSCfi/beacon-python/blob/master/beacon_api/schemas/query.json.

### PR DESCRIPTION
**Pull request by teemukataja**
Fixes #2 

Also fixes some `int` and `bool` conversions to APIs.

~~Missing feature:~~
- ~~Disable `Variant Type` if `Alternate Bases` has been filled.~~

### CODE REVIEW

File: network_apis/utils/utils.py [hunk 1]
```diff
@@ -176,26 +177,27 @@
...
-        return True if value else False
-    else:
-        # value is a string
-        if value.lower() in truthy:
-            return True
-        elif value.lower() in falsy:
-            return False
-        else:
-            raise ValueError(f'Value should be one of {truthy+falsy}')
+# Superceded by from 'distutils.util import strtobool'
```
Comment (blankdots) [inline network_apis/utils/utils.py]: we can remove this code if it is superseded.

Comment (teemukataja) [reply network_apis/utils/utils.py]: Done

File: ui/ui/view/view.js [hunk 2]
```diff
@@ -84,32 +103,175 @@ angular.module('beaconApp.view', ['ngRoute', 'ngMaterial', 'ngMessages', 'ngCook
     }
   }
 
+  // Exlusive or (XOR) manager for coordinates and variant transformation
```
[the remark this answers is not in the document]
Comment (teemukataja) [reply ui/ui/view/view.js]: I agree, it is not pretty

File: ui/ui/view/view.html [hunk 3]
```diff
@@ -47,13 +47,158 @@
               </md-input-container>
             </div>
           </form>
+          <form name="advancedSearchForm" ng-submit="submit('advanced')" class="form-inline">
```
[the remark this answers is not in the document]
Comment (teemukataja) [reply ui/ui/view/view.html]: Agreed to have it like this for now (prototype anyway) due to some angular-md priorities..

**Pull request merged**